src/SME.SGP.Aplicacao/Interfaces/CasosDeUso/Aula/IExcluirAulaWorkerUseCase.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SME.SGP.Infra;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SME.SGP.Aplicacao.Interfaces.CasosDeUso.Aula
```

Comment:
```suggestion
namespace SME.SGP.Aplicacao.Interfaces
```
Não estamos usando o caminho completo no namespace, apenas o caminho do modulo, no maximo um .Interfaces